__tests__/app.test.js

```diff
@@ -67,6 +67,7 @@ describe('/api/articles/:article_id', () => {
           votes: expect.any(Number),
           article_img_url: expect.any(String),
           body: expect.any(String),
+          comment_count: expect.any(Number)
```

Comment:
Ideally, I wouldn't just have a number here, I'd have the right number 😄 Can you update this to check for the correct number of comments?